src/components/Hovercard/Hovercard.test.tsx

```diff
@@ -182,7 +182,7 @@ describe('<Hovercard />', () => {
 
       it('the Hovercard closes after a delay', () => {
         jest.runTimersToTime(500);
-        expect(component.find(FabricCallout).length).toBe(0);
```

Comment:
I'd delete all these tests. expect(component).toMatchSnapshot(); means we dont need to be poking around like this